Sources/StreamChat/Database/DTOs/MessageDTO.swift

```diff
@@ -42,7 +42,8 @@ class MessageDTO: NSManagedObject {
     @NSManaged var quotedMessage: MessageDTO?
     @NSManaged var quotedBy: Set<MessageDTO>
     @NSManaged var searches: Set<MessageSearchQueryDTO>
-    
+    @NSManaged var previewIn: ChannelDTO?
```

Comment:
Maybe a little bit easier to read?
```suggestion
    @NSManaged var previewFromChannel: ChannelDTO?
```

or `previewOfChannel`

Reply:
Renamed ✅